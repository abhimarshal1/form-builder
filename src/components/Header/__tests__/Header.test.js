import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

// Components
import { PlaygroundHeader } from '..';

const renderer = new ShallowRenderer();

describe('PlaygroundHeader', () => {
  test('shallow render & match the snapshot', () => {
    renderer.render(<PlaygroundHeader className="playground-header" />);

    const output = renderer.getRenderOutput();
    expect(output).toMatchSnapshot();
  });
});
