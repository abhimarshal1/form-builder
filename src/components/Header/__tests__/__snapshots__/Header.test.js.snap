// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlaygroundHeader shallow render & match the snapshot 1`] = `
<styled__StyledHeader
  className="playground-header"
>
  <Header
    as="h2"
  >
    Playground
  </Header>
</styled__StyledHeader>
`;
